Disabled cursor visibility and removed some comments

Taqi opened the pull request.

jmcduff commented on Assets/Prefabs/Player/Main Player.prefab:
```diff
@@ -87,7 +87,7 @@ MonoBehaviour:
   m_EditorClassIdentifier: 
   MinForwardSpeed: 6
-  MaxForwardSpeed: 9
+  MaxForwardSpeed: 11
```
goodbye 69 u will be missed

danielwiktorczyk reviewed (approved):
LET'S GO GO GO

Taqi committed:
Removed cursor during game play, adjusted audio, and cleaned-up code


Re-enabled cursor on main menu screen and pause menu


Changed speed of player


Added fish type for the large fishes


Boost eat sfx audio levels and prepare project for build
